README.md

```diff
@@ -28,6 +28,9 @@ The attribute naming must be identical to the naming in brevo
 The following variables must be set manually
 
 -   `BREVO_API_KEY`
+-   `BREVO_TEMPLATE_DOUBLE_OPT_IN`
+-   `ECG_RTR_LIST_API_KEY`
+-   `BREVO_ALLOWED_REDIRECT_URL` (set to `SITE_URL` per default)
```

Comment:
the brevo allowed redirect url is set in the .env, or why would we need to set it manually?

Reply:
Good point, this is not needed for the demo 
removed in [f269a7c](https://github.com/vivid-planet/comet-brevo-module/pull/7/commits/f269a7c631f623c134b51974525329f7666a3793)